### ISSUE TITLE
Readme should advise on how to distinguish CP210{1..4}

### ISSUE THREAD

**Issue by mk-pmb** [VCTLabs/cp210x-program#37]
First, thanks for your effort!
The tables with the vendor and product IDs all list CP210{1..4} the same, so the readme should advise on how to distinguish them. Even a negative acknowledgement like

> There's currently no known way to distinguish them via software, so you have to rely on product descriptions.

would help the reader save time in research.

**Comment by adminy** [comment]
when you do `lsusb`, you get something like this:
```
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 003 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
Bus 003 Device 008: ID 10c4:ea60 Silicon Labs CP210x UART Bridge
Bus 003 Device 009: ID 0bda:2838 Realtek Semiconductor Corp. RTL2838 DVB-T
Bus 004 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
```
and I suppose 10c4:ea60 is a specific model, e.g. CP2104

**Comment by mk-pmb** [comment]
Yes, my lsusb has the exact same entry, `ID 10c4:ea60 Silicon Labs CP210x UART Bridge`, so maybe that `get_part_number` could work for me. How do I use that?
